sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -0,0 +1,777 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+# pylint: disable=protected-access
+
+import re
+from collections import namedtuple
+from enum import Enum
+from ._helpers import get_field_scalar_value
+
+
+PageRange = namedtuple("PageRange", "first_page last_page")
+Point = namedtuple("Point", "x y")
+
+
+def get_elements(field, ocr_result):
+    elements = []
+    try:
+        for item in field.elements:
+            extracted_word = None
+            nums = [int(s) for s in re.findall(r'\d+', item)]
+            read = nums[0]
+            line = nums[1]
+            if len(nums) == 3:
+                word = nums[2]
+                ocr_word = ocr_result[read].lines[line].words[word]
+                extracted_word = ExtractedWord._from_generated(ocr_word, page=read+1)
```

Comment:
Will an `extracted_word` ever be `False`:y? If not, why are we also extracting the line? We could move the later `if extracted_word: elements.append(...)` here...

Reply:
Fixed the logic here